src/ape/utils.py

```diff
@@ -609,6 +611,37 @@ def provider(self) -> "ProviderAPI":
             raise ProviderNotConnectedError()
         return self.network_manager.active_provider
 
+    def create_contract_container(self, contract_type: "ContractType") -> "ContractContainer":
+        """
+        Helper method for creating a ``ContractContainer``.
+
+        Args:
+            contract_type (``ContractType``): Type of contract for the container
+
+        Returns:
+            :class:`~ape.contracts.ContractContainer`
+        """
+        from ape.contracts.base import ContractContainer
```

Comment:
We could make two cached properties for importing of `ContractContainer` and `ContractInstance`...
However, I suspect `python` does this already automatically. I have noticed that the second time you import something in a python session is a lot faster than the first.

Reply:
yeah, imports are cached